src/l2/input_parser.star

```diff
@@ -150,6 +153,30 @@ def _parse_instance(l2_args, l2_name, l2_id_generator, registry):
         registry=registry,
     )
 
+    # We add the flashblocks params
+    l2_params["flashblocks_websocket_proxy_params"] = _flashblocks_input_parser.parse(
+        websocket_proxy_args=l2_params["flashblocks_websocket_proxy_params"],
+        network_params=l2_params["network_params"],
+        registry=registry,
+    )
+
+    # WARNING: We can't have another participant with the same name
+    # TODO: We should check whether any participant has the same name
+    # as the flashblocks RPC participant and fail if so
+    # Add the image if it's not already set
+    if l2_params["flashblocks_rpc_params"]:
+        if not l2_params["flashblocks_rpc_params"].get("image"):
```

Comment:
Not a big fan of this logic that pertains to flashblocks being here. The way to go would be to create a small inout parser that would just do this bit, then it would use the EL parser like you're doing below. That way the flashblocks logic is encapsulated there